src/main/java/com/example/bookmyshow/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.bookmyshow.controller;
+
+import com.example.bookmyshow.auth.Token;
+import com.example.bookmyshow.model.User;
+import com.example.bookmyshow.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/users")
+@CrossOrigin(origins = "http://localhost:5173")
```

Comment:
Seems in every controller we have to add this. Please find a better approach to fix cors

Reply:
I am changing it